CONTRIBUTING.md

```
@@ -0,0 +1,540 @@
+# Contributing to secure-uploads
+
+As an open-source project, the `artisanpack-ui/secure-uploads` package is open to contributions from everyone. You don't need to be a developer to contribute. Whether it's writing code, improving documentation, exercising the package against your own Laravel app and reporting bugs, or helping triage issues, there's a place for you here.
+
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [Ways to Contribute](#ways-to-contribute)
+- [Getting Started](#getting-started)
+- [Issue Templates](#issue-templates)
+- [Branching Strategy](#branching-strategy)
+- [Pull Request Process](#merge-request-process)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)